gc/base/GCExtensionsBase.hpp

```diff
@@ -293,6 +293,8 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	bool doOutOfLineAllocationTrace;
 	bool doFrequentObjectAllocationSampling; /**< Whether to track object allocations*/
 	uintptr_t oolObjectSamplingBytesGranularity; /**< How often (in bytes) we do an ool allocation trace */
+	uintptr_t objectSamplingBytesGranularity; /**< How often (in bytes) we do an allocation trace (for triggering J9HOOK_MM_OBJECT_ALLOCATION_SAMPLING) */
+
```

Comment:
you should not really be referring to J9 specific hook. Perhaps you can say:
How often (in bytes) we do allocation sampling as tracked by per thread's local _traceAllocationBytes. 
And similarly adjust the OOL one.